src/constants/whitelisted-dapps.ts

```diff
@@ -0,0 +1,16 @@
+// Maps whitelisted dapps to their ENS names
+export const whitelistedDapps: Array<{ url: string; dappId: string }> = [
+  { url: 'https://fairdrive.dev.fairdatasociety.org', dappId: 'fairdrive-dev' },
+  { url: 'https://fairdrive.fairdatasociety.org', dappId: 'fairdrive' },
+  { url: 'https://fairdrive.dev.fairdatasociety.org/apps/consents', dappId: 'consents-dev' },
+  { url: 'https://fairdrive.fairdatasociety.org/apps/consents', dappId: 'consents' },
+  { url: 'https://app.photo.dev.fairdatasociety.org', dappId: 'photo-album-dev' },
+  { url: 'https://app.photo.fairdatasociety.org', dappId: 'photo-album' },
+  { url: 'https://app.dracula.dev.fairdatasociety.org', dappId: 'dracula-dev' },
+  { url: 'https://app.dracula.fairdatasociety.org', dappId: 'dracula' },
+  { url: 'https://fairdrive.dev.fairdatasociety.org/apps/dracula', dappId: 'dracula-dev' },
+  { url: 'https://fairdrive.fairdatasociety.org/apps/dracula', dappId: 'dracula' },
+  { url: 'https://fairdrive.dev.fairdatasociety.org/apps/slidezz', dappId: 'slidezz-dev' },
+  { url: 'https://fairdrive.fairdatasociety.org/apps/slidezz', dappId: 'slidezz' },
+  { url: 'http://localhost:3000', dappId: 'fairdrive' },
```

Comment:
any application can run under `localhost:3000`, and it raises security concern. I guess you added this for testing FairDrive for which I would suggest another approach.

Reply:
I thought localhost will be used only by developers, but yes, it's better to remove it.